pytest_pyvista/pytest_pyvista.py

```diff
@@ -241,4 +245,8 @@ def verify_image_cache(request, pytestconfig):
     )
     pyvista.global_theme.before_close_callback = verify_image_cache
 
+    def reset():
+        pyvista.global_theme.before_close_callback = None
```

Comment:
I'm wondering if we should additionally do `pyvista.close_all` in `reset` to help downstream libs handle cleanup.  However, it might give confusing results if they use this fixture in some tests but not others.  Maybe this warrants a separate fixture here like `auto_close`.  We could use a flag to turn it on and off.  This should be a separate PR as it adds new features if implemented this way.